packages/usa-radio/src/usa-radio--tile.twig

```diff
@@ -2,15 +2,21 @@
   <fieldset class="usa-fieldset">
     <legend class="usa-legend">Select one historical figure</legend>
     <div class="usa-radio">
-      <input class="usa-radio__input usa-radio__input--tile" id="historical-truth-2" type="radio" name="historical-figures-2" value="sojourner-truth" checked {% if aria_disabled == true %}aria-disabled="true"{% endif %}>
+      <input class="usa-radio__input usa-radio__input--tile" id="historical-truth-2" type="radio" name="historical-figures-2" value="sojourner-truth" checked 
```

Comment:
There are a lot of attributes in these radios, maybe we can increase readability by separating them out into their own line.

```suggestion
<input 
  class="usa-radio__input usa-radio__input--tile" 
  id="historical-truth-2" 
  type="radio" 
  name="historical-figures-2" 
  value="sojourner-truth" 
  checked 
  {% if disabled_state == "disabled" %}disabled{% endif %} 
  {% if disabled_state == "aria_disabled" %}aria-disabled="true"{% endif %}
>
```

Reply:
Resolved in [2afdb02](https://github.com/uswds/uswds/pull/5063/commits/2afdb022226a20eac994818285486a1ad4b7bfef)